runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -864,14 +906,19 @@ const onert::ir::Graph *nnfw_session::primary_subgraph()
 {
   if (_subgraphs)
   {
-    assert(!_execution);
+    assert(!(_execution || _executions.size()));
```

Comment:
How about this?

`!(_execution || _executions.empty())`

Reply:
That's a good point. 
do you mean "!(_execution || !_executions.empty())"?

Reply:
sorry.  you are right... 

So i think..

`(!_execution && _executions.empty())`

This looks good for readability.